src/openassetio-python/package/openassetio/constants.py

```diff
@@ -23,18 +23,13 @@
 
 kSupportedAttributeTypes = (str, int, float, bool, type(None))
 
-## @name Field Names
-## Field names are to be used whenever data is get or set from a dictionary by
-## key, rather than through an accessor in some wrapper class (eg:
-## Item/Specification etc...). Bare strings should never be used to help
-## protect against inconsistencies and future changes.
+## @name Info dict field Names
+##
+## Bare strings should never be used to help protect against
+## inconsistencies and future changes.
 ##
-## These fields should also be used, if necessary, when declaring derived
-## classes of a @ref Specification.
 ## @{
 
-kField_ItemType = "fnItemType"
-
 # General
 
 kField_DisplayName = "displayName"
```

Comment:
[question] What is the purpose of the `"displayName"` field? 

Is it just a convenient alternative to calling `manager.displayName()` if you already happen to have the `.info()` dictionary?  

Although in that case, the host would always have to code defensively in case `"displayName"` wasn't in `.info()`, and so fall back to calling `.displayName()` anyway, kinda defeating the point.

Reply:
Yeah, this one one of the "use needs validating" ones - I _think_ it was as before we refactord [this to c++](https://github.com/OpenAssetIO/OpenAssetIO/blob/db3ddc2dc5bee475dfa9c4388bf25854f479174d/src/openassetio-core/src/hostApi/ManagerFactory.cpp#L62), the Python version used to set that key with the result of `displayName` in the returned dict, before we made it a proper struct. This can most likely go.

Reply:
Display name? [What display name?](https://github.com/OpenAssetIO/OpenAssetIO/pull/1001/commits/a4f10c1b4169ccbc3d86a2c240effdbbc47a6cd3)